pkgs/development/python-modules/qutip/default.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv
+, lib
```

Comment:
Only use stdenv and stdenv.lib in all other places.

Reply:
Hi Sandro,

First-time contributor here. I take that to mean that `lib` is unused? Thanks for pointing that out! I admit to roughly pattern-matching against other pythonPackages, and easily could have done a thing or two without knowing why.

While you're taking a look at this PR, I have two other questions if that's all right:
1. The bot `grahamcofborg-eval` failed with the message, "This PR does not cleanly list package outputs after merging." What does that mean? What can I do to fix it?
2. It appears that this bot also doesn't like the maintainer name (`mcncm`, _i.e._ me) isn't in scope. Should it be an attribute of `maintainers`? Why is this building successfully for me if there's an undefined variable?

Thank you!

Reply:
> I take that to mean that lib is unused?

It is not really unused but the one place it is used it can easily be replaced with ``stdenv.lib``. Tbh it is just a nitpick to have less inputs. It does not really change anything functional wise.

> It appears that this bot also doesn't like the maintainer name (mcncm, i.e. me) isn't in scope. Should it be an attribute of maintainers? Why is this building successfully for me if there's an undefined variable?

You need to add yourself to https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix. The contributing guide requires to do that in a separate commit. The commit message should look like ``maintainers: add name``.

> The bot grahamcofborg-eval failed with the message, "This PR does not cleanly list package outputs after merging." What does that mean? What can I do to fix it?

If you follow the details link you can read the log which complains about the missing maintainer.

```
undefined variable 'mcncm' at 
```

https://gist.github.com/GrahamcOfBorg/02f8ab994b78fc43abd018687bec154c

Reply:
Hi Sandro,

Thanks for the elaboration. I've removed the dependence on `lib`, added myself as a maintainer, and modified the derivation slightly to add openmp support to qutip.